package.json

```diff
@@ -51,7 +53,7 @@
     "patch-package": "^6.5.1",
     "prettier": "^2.8.6",
     "ts-node": "^10.9.1",
-    "typescript": "^5.0.2"
+    "typescript": "^4.9.4"
```

Comment:
`expo install --fix` wanted to lower this again. I'll wait to upgrade until expo likes 5.0.